src/content/en/fundamentals/engage-and-retain/push-notifications/permissions-subscriptions.markdown

```diff
@@ -72,9 +72,8 @@ explains exactly why you want to send them notifications.
 If the user approves, we need to send a subscription request to the push
 manager. Do this by calling `PushManager.subscribe()` (lines 5 through 9). In
 this example, we're passing it an object with  `userVisibleOnly` set to `true` so
-that all push messages  sent the client will be shown to the user as a
-notification. We're also  including an `applicationServerKey` converted to an
-integer array.
+that all push messages sent to the client will be shown to the user as a
```

Comment:
This feels like an awkward phrasing. It's required by browsers and means that you must show a notification for every push message.